src/main/java/org/stocker/PingVerticle.java

```diff
@@ -78,12 +79,12 @@ public void start() {
 
       Router router = new Router();
       router.get("/healthCheck", new HealthCheck());
-      router.post("/update/stocks/:date", updateRecordsRoute);
-      router.get("/stock/:stock/date/:date", getStockRoute);
-      router.get("/prediction/stock/:stock/type/:type", getPredictionTemplatedRoute);
-      router.get("/prediction/stock/:stock", getPredictionRoute);
-      router.get("/data/stock/:stock/type/:type", getAggregatedDataTemplatedRoute);
-      router.get("/data/stock/:stock", getAggregatedDataRoute);
+      router.post(":market/update/stocks/:date", updateRecordsRoute);
```

Comment:
also , a more flexible way would be take in market as a query param . So if one isnt provided , default to US . That way existing endpoints just default to US .